packages/squidex/src/errors/index.ts

```diff
@@ -0,0 +1,15 @@
+export class SquidexError extends Error {
+  constructor(public cause?: unknown) {
```

Comment:
`cause` of type unknown seems a little..unnecessary. Since SquidexErrors are all results of an HTTP call shouldnt this be named `response` and given a more appropriate type?

Reply:
No - because this could be either http response or whole unknown error.
Like JSON parsing error - for example